.github/workflows/run_task.yml

```diff
@@ -49,4 +54,4 @@ jobs:
           aws_vpc_security_groups: |
             sg-0de7ca06bb5972dcb
           repo_name: "newrelic/fluent-bit-package"
-          ref: ${{ env.GIT_BRANCH }}
\ No newline at end of file
+          ref: ${{ inputs.ref == '' && env.GIT_BRANCH || inputs.ref }}
```

Comment:
Why do we need this `ref` for? Weren't we already checking out the branch we're interested in in the Fargate container?